### PR TITLE
[13.0][MIG] sale_product_set

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of module sale_product_set to v13

 cc ~ @ForgeFlow

### CODE REVIEW

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Skip Existing Products flag is apparently not working

**Review by rousseldenis — APPROVED** [review]
Code review.

Another great module passing versions since v8!

**Comment by rousseldenis** [comment]
@AdriaGForgeFlow Could you attend @HviorForgeFlow comment ?

File: sale_product_set/wizard/product_set_add.py [hunk 1]
```diff
@@ -22,9 +20,7 @@ class ProductSetAdd(models.TransientModel):
     product_set_id = fields.Many2one(
         "product.set", "Product set", required=True, ondelete="cascade"
     )
-    quantity = fields.Float(
-        digits=dp.get_precision("Product Unit of Measure"), required=True, default=1
-    )
+    quantity = fields.Float(digits="Product Unit of Measure", required=True, default=1)
```
Comment (nikul-serpentcs) [inline sale_product_set/wizard/product_set_add.py]: @AdriaGForgeFlow  `default=1` -> ` default=1.0`

Comment (AdriaGForgeFlow) [reply sale_product_set/wizard/product_set_add.py]: Done :+1:

**Review by HviorForgeFlow — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Pull request merged**